Jenkinsfile

```diff
@@ -169,41 +192,38 @@ pipeline {
                             deleteDir()
                             unstash 'CIScripts'
                             powershell script: './CIScripts/Test.ps1'
+
+                            // TODO: you can do better
+                            stash name: 'testReport', includes: 'CIScripts/Test/testReport.xml'
                         }
                     }
                 }
             }
-        }
-    }
 
-    environment {
-        LOG_SERVER = "logs.opencontrail.org"
-        LOG_SERVER_USER = "zuul-win"
-        LOG_ROOT_DIR = "/var/www/logs/winci"
-        BUILD_SPECIFIC_DIR = "${ZUUL_UUID}"
-        JOB_SUBPATH = env.JOB_NAME.replaceAll("/", "/job/")
-        REMOTE_DST_FILE = "${LOG_ROOT_DIR}/${BUILD_SPECIFIC_DIR}/log.txt.gz"
+            post {
```

Comment:
This additional nested post stage seems unnecessary. Could you explain why is it here?